.github/workflows/release-drafter.yml

```diff
@@ -0,0 +1,24 @@
+name: Release Drafter
+
+on:
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: read
+
+jobs:
+  update_release_draft:
+    if: github.repository == 'dask/dask'
+    permissions:
+      # Write permission is required to create a GitHub release
+      contents: write
```

Comment:
It makes sense why this is needed, but was a little concerning at first. Though since this is only gated to pushed to `dask/dask` `main`, it seems fine to me

Reply:
Yup exactly. This only runs on pushes to `dask/dask@main`.